src/renderer.ts

```diff
@@ -104,14 +104,24 @@ export class DefaultRenderer implements Renderer {
    * });
    * ```
    */
+  underMeanColor: string
+  overMeanColor: string
+
+   constructor(underMeanColor = "#0000ff", overMeanColor = "#ff0000") {
+    super()
+
+    this.underMeanColor = underMeanColor
+    this.overMeanColor = overMeanColor
+  }
+
   public render(
     { count, position }: Cluster,
     stats: ClusterStats,
     map: google.maps.Map
   ): Marker {
     // change color if this cluster has more markers than the mean cluster
     const color =
-      count > Math.max(10, stats.clusters.markers.mean) ? "#ff0000" : "#0000ff";
+      count > Math.max(10, stats.clusters.markers.mean) ? this.overMeanColor : this.underMeanColor;
```

Comment:
```suggestion
      count > Math.max(10, stats.clusters.markers.mean) ? var(--over-mean) : var(--under-mean);
```

Wondering if it would be easier to just change the hex values to css variables.

Reply:
@behunin they didn't accept my original pull request, not sure what to do to make them look at this issue :/

Reply:
@jpoehnelt  Could you poke whomever google has assigned to this product right now? How can we get some traction here?